takeshape/penny
# Handle customers with inactive accounts during sign in / creation / password reset

<img width="654" alt="Screenshot 2023-05-03 at 14 05 50" src="https://user-images.githubusercontent.com/277401/236022162-8d8a3a02-1efe-4e0b-b13a-aeaa2e299029.png">

### Test Plan (Steps to test):

1. Attempt to sign in with a known disabled email — `michael+earth1@takeshape.io` works — confirm you get the activation email
1. Same as above for creation
1. Same as above for reset password
2. Now try all three of those with a know active email / account
3. Try resetting the password of an email that doesn't exist
4. Try creating an account that already exists
